Octokit/Models/Response/CompareResult.cs

```diff
@@ -8,18 +8,18 @@ namespace Octokit
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public class CompareResult
     {
-        public string Url { get; set; }
-        public string HtmlUrl { get; set; }
-        public string PermalinkUrl { get; set; }
-        public string DiffUrl { get; set; }
-        public string PatchUrl { get; set; }
-        public GitHubCommit BaseCommit { get; set; }
-        public GitHubCommit MergedBaseCommit { get; set; }
-        public string Status { get; set; }
-        public int AheadBy { get; set; }
-        public int BehindBy { get; set; }
-        public int TotalCommits { get; set; }
-        public IReadOnlyCollection<GitHubCommit> Commits { get; set; }
+        public string Url { get; protected set; }
+        public string HtmlUrl { get; protected set; }
+        public string PermalinkUrl { get; protected set; }
+        public string DiffUrl { get; protected set; }
+        public string PatchUrl { get; protected set; }
+        public GitHubCommit BaseCommit { get; protected set; }
+        public GitHubCommit MergedBaseCommit { get; protected set; }
+        public string Status { get; protected set; }
+        public int AheadBy { get; protected set; }
+        public int BehindBy { get; protected set; }
+        public int TotalCommits { get; protected set; }
+        public IReadOnlyCollection<GitHubCommit> Commits { get; protected set; }
```

Comment:
Maybe it's a good idea to standardize on either `IReadOnlyList<T>` or `IReadOnlyCollection<T>`?